_config.yml

```diff
@@ -134,7 +143,7 @@ content:
           fr: "diminuant doucement"
       TIM:
         name:
-          en: "Block time:"
+          en: "Disinflationary supply (~18 million)"
```

Comment:
may I suggest being a little more precise and using `~18.4 million`?

Reply:
Ok, no problem